setup-tests.js

```diff
@@ -0,0 +1,5 @@
+import { jestPreviewConfigure } from 'jest-preview';
+
+if (!process.env.CI) {
```

Comment:
The idea is that we don't need to run the `autoPreview` mode in CI but just locally.